### DISCUSSION TITLE
Please tell me about TTGO TDISPLAY1.14

### DISCUSSION THREAD

**Discussion by Aron-LIU66** [BitMaker-hub/NerdMiner_v2#174] [Q&A]
Hello: I use ESP Tool with bin LILYGO TDisplay v1.14
But my TTGO TDISPLAY1.14 always has no computing power
What's the problem? Thank you
![379102912_6875191409192963_4963170354073732666_n](https://github.com/BitMaker-hub/NerdMiner_v2/assets/145151538/382963ec-0623-432e-a413-e0b31932768a)

**Comment by BitMaker-hub** [comment]
I suggest you to recheck your config. If introduced Wireless config is ok, and BTC address too. 
If still same, ask telegram group
